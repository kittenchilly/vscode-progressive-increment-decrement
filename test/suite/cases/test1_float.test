##skipFirstNumber
abcdef0ghilm
abcdef0ghilm
abcdef0ghilm
abcdef0ghilm
abcdef0ghilm
abcdef0ghilm
abcdef0ghilm
abcdef0ghilm
abcdef0ghilm
abcdef0ghilm