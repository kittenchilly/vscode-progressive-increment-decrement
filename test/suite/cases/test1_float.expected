##skipFirstNumber
abcdef0ghilm
abcdef0.2ghilm
abcdef0.4ghilm
abcdef0.6ghilm
abcdef0.8ghilm
abcdef1ghilm
abcdef1.2ghilm
abcdef1.4ghilm
abcdef1.6ghilm
abcdef1.8ghilm